modules/nw-ingress-view.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-view-{context}"]
+= View the default ingress controller
+
+The Ingress Operator is a core feature of OpenShift and is enabled out of the
```

Comment:
Should `OpenShift` be {product-title} here?

Reply:
It should, will fix - thanks!